libs/specialPushNotificationManager.js

```diff
@@ -0,0 +1,92 @@
+var sprintf = require('sprintf-js');
+
+var db, queue, pushNotifications, done, habitrpgUsers, notificationBuckets, timezoneQuery;
+
+var pageLimit = 30;
+
+function processUsersWithDevices(users) {
```

Comment:
Can you document your process here. There is a lot of assumed knowledge. I believe you are just randomly placing users in buckets right?